src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,58 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_LENGTH = 5;
+    private int size;
+    private K[] keyArray;
+    private V[] valueArray;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```
let's not use such a general names as array/list/queue/map etc in our namings